content/800-data-platform/050-data-proxy/index.mdx

```diff
@@ -0,0 +1,103 @@
+tocDepth: 2
+---
+
+<TopBlock>
+
+Serverless functions are ephemeral and short-lived, so their database connections are numerous and brief.
+
+Because of this, using traditional databases in serverless environments often leads to the exhaustion of the maximum number of concurrent database connections and increased latencies when each request needs to establish a database connection.
+
+Data Proxy in the Prisma Data Platform provides database connection management and pooling, load balancing, scaling, and fault tolerance so that you can manage and scale database connections for applications that run with Prisma ORM in serverless environments. With Data Proxy, your application can scale and maintain predictable database performance as it also limits the total number of database connections.
+
+Designed specifically with serverless functions in mind, Data Proxy provides a wide range of benefits for applications that run in serverless and edge environments.
+
+* [Architecture](/data-platform/data-proxy#architecture)
+* [Connection pooling and connection management](/data-platform/data-proxy#connection-pooling-and-connection-management)
+* [Reduced bundle size](/data-platform/data-proxy#reduced-bundle-size)
+* [Cold starts prevention](/data-platform/data-proxy#cold-starts-prevention)
+
+</TopBlock>
+
+## Architecture
+
+Prisma develops and maintains the Data Proxy infrastructure and services to provide connection pooling, load balancing, scaling, and fault tolerance for database applications that run in serverless and edge environments.
+
+Data Proxy infrastructure is currently available in two regions:
+
+* US East 1 (West Virginia, United States)
+* EU Central 1 (Frankfurt, Germany)
+
+<Admonition>
+
+**Request a new Data Proxy region**<br /><br />
+
+If you want us to make Data Proxy available in a new region, you can submit your request [here](
+https://prisma103696.typeform.com/to/Y03RBE5c).
+
+</Admonition>
+
+In each region, the Data Proxy infrastructure has multiple clusters that enable the load balancing, high availability, and failover of database connection pooling and management.
+
+You can use Data Proxy with [all databases that Prisma ORM supports](/reference/database-reference/supported-databases).
+
+## Connection pooling and connection management
+
+Data Proxy provides connection pooling so that your application can reuse active connections and never exhaust the maximum number of connections that your database allows.
+
+Connection pooling with the Data Proxy utilizes orchestrated Prisma [query engine](/concepts/components/prisma-engines/query-engine) technology to open, maintain, and close connection pools and, thus, help applications load-balance and scale database requests based on demand.
```

Comment:
```suggestion
Connection pooling with the Data Proxy utilizes orchestrated Prisma [query engine](/concepts/components/prisma-engines/query-engine) technology to open, maintain, and close connection pools. It helps applications load-balance and scale database requests based on demand.
```

I like a good "thus", but I think it's probably best avoided in tech docs, and also this sentence is better if split.